ci-operator/step-registry/ovn/sdn-migration-rollback/ovn-sdn-migration-rollback-commands.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TARGET=${TARGET:-OpenShiftSDN}
+
+oc patch MachineConfigPool master --type='merge' --patch '{"spec":{"paused":true}}'
+oc patch MachineConfigPool worker --type='merge' --patch '{"spec":{"paused":true}}'
+
+# Reset the spec.migration before we can set it to other value
+oc patch Network.operator.openshift.io cluster --type='merge' --patch '{"spec":{"migration":null}}'
+# Wait until CNO update the applied-cluster cm to the latest
+sleep 10
```

Comment:
```suggestion
echo "waiting for the changes to roll out"
oc wait --for=condition=Updating=false mcp/worker --timeout=600s
oc wait --for=condition=Updating=false mcp/master --timeout=600s
echo "done waiting"
```

Reply:
gotcha. I'd prefer to eradicate any sleeps and use `oc wait` (for network.config and such) as much as possible

Reply:
I agree on this. Avoiding sleeps and polling on something is best practice. some of these sleeps are minutes long. What if the wait only needs to be 5 seconds in some cases, then we waste lots of time. There's usually always a way to poll and avoid a sleep.

Reply:
fixed